internal/util/k8s/parameters.go

```diff
@@ -100,3 +107,15 @@ func GetSnapshotMetadata(parameters map[string]string) map[string]string {
 
 	return newParam
 }
+
+// GetCOKeyName returns the variable name that is used to set CO name
+// via env variables.
+func GetCOKeyName() string {
+	return coKeyName
+}
+
+// GetCOClusterName reads the `CO_CLUSTER_NAME` from the
+// environment and returns its value.
+func GetCOClusterName() string {
+	return os.Getenv(coName)
```

Comment:
it is way cleaner to pass this as a commandline option instead of a new (and hard to find documentation about) environment variable

Reply:
Sure